src/routes/api/index.router.js

```diff
@@ -1,8 +1,9 @@
 import express from 'express';
+
 const router = express.Router();
 
 router.get('/', (req, res) => {
-    res.status(200).json({ Message: 'Welcome! This is the NorthStar Barefoot Nomad homepage' });
+  res.status(200).json({ Message: 'Welcome! This is the NorthStar Barefoot Nomad homepage' });
```

Comment:
You might want to install prettier for improved formatting.

Reply:
I think I have it. But sometimes I checkout to develop and 4-space tabs everywhere. I run ESLint to correct them.

Reply:
You can also correct the indentation of that workspace.
But it's fine. The code would have look cleaner like so:

```
 res.status(200)
.json({ 
     Message: 'Welcome! This is the NorthStar Barefoot Nomad homepage' 
    });
```

I just noticed that this isn't even your code. My bad!